scripts/fetch_smugmug_activity.py

```diff
@@ -170,40 +144,39 @@ def insert_comments(connection, new_comments):
 
     cursor = connection.cursor()
     query = '''
-        insert into smugmug_comments
-            (comment_id, comment_url, thumbnail_url, author, comment, created_at)
-        values
-            (%(comment_id)s, %(comment_url)s, %(thumbnail_url)s, %(author)s, %(comment)s, %(created_at)s)
+        insert into smugmug_comments (
+            comment_id,
+            comment_url,
+            thumbnail_url,
+            author,
+            comment,
+            created_at
+        )
+        values (
+            %(comment_id)s,
+            %(comment_url)s,
+            %(thumbnail_url)s,
+            %(author)s,
+            %(comment)s,
+            %(created_at)s
+        )
+        on duplicate key update
+            comment_url = values(comment_url),
+            thumbnail_url = values(thumbnail_url),
+            author = values(author),
+            comment = values(comment)
         '''
     cursor.executemany(query, new_comments)
     connection.commit()
     cursor.close()
 
 
-def get_most_recent_photo(connection):
-    cursor = connection.cursor()
-    query = '''
-        select gallery_slug, photo_id, created_at
-        from smugmug_uploads
-        order by id desc
-        limit 1
-        '''
-
-    cursor.execute(query)
-    for (gallery_slug, photo_id, created_at) in cursor:
-        return (gallery_slug, photo_id, created_at)
-
-    return (None, None, datetime.min)
-
-
-def fetch_recent_photos(last_gallery_slug, last_photo_id, last_photo_uploaded_at):
+def fetch_recent_photos():
     url = PHOTO_FEED_URL
     recent_photos = []
     seen_photos = set()
 
-    found_last_photo = False
-    while not found_last_photo:
-
+    while True:
```

Comment:
If you could add a brief comment that we need this to deal with pagination of the feed, that'd unconfuse me :)